pkgs/top-level/python-packages.nix

```
@@ -11041,9 +11039,7 @@ self: super: with self; {
 
   qtconsole = callPackage ../development/python-modules/qtconsole { };
 
-  qtile = callPackage ../development/python-modules/qtile {
-    wlroots = pkgs.wlroots_0_15;
-  };
+  qtile = callPackage ../development/python-modules/qtile {};
```

```suggestion
  qtile = callPackage ../development/python-modules/qtile {
      wlroots = pkgs.wlroots_0_16;
  };
```
we should keep depending on wlroots_0_* as each new wlroots release comes with so many breaking changes

---

Thanks. I thought sway didn't have this but I didn't look correctly.